openzwavemqtt/models/command_class.py

```diff
@@ -23,13 +21,19 @@ def instance(self) -> int:
         return self.data.get("Instance")
 
     @property
-    def command_class_id(self) -> int:
```

Comment:
Let's not remove the ID property, as it is part of the raw data. Let's just keep both. Removing it will also trigger a warning in `instance_from_file.py` script.

Reply:
For now, I've just reverted it, we can update it as soon as as @Fishwaldo updated the value upstream in the json responses.
command_class_id will now return the enum value (which is off course also an int, so backwards compatible).